tests/test_base_device.py

```diff
@@ -991,31 +928,6 @@ def test_generate_samples(self, num_wires, num_shots):
         # nothing else is in samples
         assert [s for s in samples if not s in np.array([exp_res0, exp_res1])] == []
 
-    @pytest.mark.skip(
-        reason="Tracking shot information will be addressed in the PR about options handling"
-    )
-    def test_tape_shots_used_for_estimator(self, mocker):
```

Comment:
This has been replaced with the test at Line 835. I don't know why it's misaligned, maybe due to fixing merge conflicts.